js/ad_manager_vast.js

```diff
@@ -1023,6 +1291,9 @@ OO.Ads.manager(function(_, $) {
         this.checkCompanionAds(adWrapper.ad);
         initSkipAdOffset(adWrapper);
       }
+
+      // try and ping tracking URLs
+      _handleTrackingUrls(adWrapper, ["creativeView", "start", "impression"]);
```

Comment:
Can we achieve a better granularity so that we don't fire these 3 events at the same time? It's ok if we can't, just asking if it is possible.